src/EnergyPlus/DataSizing.hh

```diff
@@ -639,24 +639,29 @@ namespace DataSizing {
     struct ZoneEqSizingData // data saved from zone eq component sizing and passed to subcomponents
     {
         // Members
-        Real64 AirVolFlow;           // design air vol flow rate for zone equipment unit [m3/s]
-        Real64 MaxHWVolFlow;         // design Hot Water vol flow for zone equipment unit [m3/s]
-        Real64 MaxCWVolFlow;         // design Cold Water vol flow for zone equipment unit [m3/s]
-        Real64 OAVolFlow;            // design outside air flow for zone equipment unit [m3/s]
-        Real64 DesCoolingLoad;       // design cooling load used for zone equipment [W]
-        Real64 DesHeatingLoad;       // design heating load used for zone equipment [W]
-        Real64 CoolingAirVolFlow;    // design cooling air vol flow rate for equipment[m3/s]
-        Real64 HeatingAirVolFlow;    // design heating air vol flow rate for equipment[m3/s]
-        Real64 SystemAirVolFlow;     // design heating air vol flow rate for equipment[m3/s]
-        bool AirFlow;                // TRUE if AirloopHVAC system air flow rate is calcualted
-        bool CoolingAirFlow;         // TRUE if AirloopHVAC system cooling air flow rate is calcualted
-        bool HeatingAirFlow;         // TRUE if AirloopHVAC system heating air flow rate is calcualted
-        bool SystemAirFlow;          // TRUE if AirloopHVAC system heating air flow rate is calcualted
-        bool Capacity;               // TRUE if AirloopHVAC system capacity is calculated
-        bool CoolingCapacity;        // TRUE if AirloopHVAC system cooling capacity is calculated
-        bool HeatingCapacity;        // TRUE if AirloopHVAC system heating capacity is calculated
-        bool SystemCapacity;         // TRUE if AirloopHVAC system heating capacity is calculated
-        bool DesignSizeFromParent;   // TRUE if design size is set by parent object - normally false, set to true for special cases e.g. ERV
+        Real64 AirVolFlow;            // design air vol flow rate for zone equipment unit [m3/s]
+        Real64 MaxHWVolFlow;          // design Hot Water vol flow for zone equipment unit [m3/s]
+        Real64 MaxCWVolFlow;          // design Cold Water vol flow for zone equipment unit [m3/s]
+        Real64 OAVolFlow;             // design outside air flow for zone equipment unit [m3/s]
+        Real64 ATMixerVolFlow;        // design ventilation air flow rate from air terminal mixer (central DOAS) [m3/s] //TRANE AT Mixer sizing
```

Comment:
We don't really need comments tagging TRANE in here.  If there's anything else to change in this PR, comments like this should go.

Reply:
I missed that one.